frontend-v2/src/styles/globals.css

```diff
@@ -28,5 +13,66 @@ body {
 @layer base {
   :root {
     --radius: 0.5rem;
+    --background: 0 0% 100%;
```

Comment:
not sure if we actually want all of these css variables here & in tw config. i need to look into shadcn config more.